### ISSUE TITLE
hard to turn off (Quit button?)

### ISSUE THREAD

**Issue by meowsbits** [etclabscore/jade-service-runner-ui#77]
I don't see a button for 'Quit' anywhere; the Service Runner isn't even listed in the Force Quit dialog... ?
![Screen Shot 2019-10-25 at 3 02 09 PM](https://user-images.githubusercontent.com/45600330/67600814-813d0580-f738-11e9-9f48-4fffed0b7d76.png)

**Comment by shanejonas** [comment]
agreed, should have a more user-friendly way to close it. I close it right now via `CMD + Q`
